src/components/user/loginForm/loginForm.scss

```
@@ -0,0 +1,12 @@
+@import '../../../styles/colors.scss';
+
+.sign-in-button {
+  margin-bottom: 15px;
+}
+
+.error {
+  font-size: 1.4rem;
+  font-weight: 600;
+  margin: 1rem 1rem;
```

This can be also be written as 
```suggestion
  margin: 1rem;
```